tally_ho/apps/tally/views/reports/staff_performance_metrics.py

```diff
@@ -172,14 +172,12 @@ def get(self, *args, **kwargs):
                 qs.values('user__username')\
                 .annotate(
                     total_forms_processed=Count('user'),
-                    total_forms_processed_with_errors=Count(
-                        'user',
-                        filter=Q(has_de_error=True)
-                    ))\
+                    total_errors=Sum(
```

Comment:
Do we need to use [Coalesce](https://docs.djangoproject.com/en/3.0/ref/models/database-functions/#coalesce) here to prevent possible `None` values returned by `Sum`?

Reply:
Done